src/main/java/kobe/Kobe.java

```diff
@@ -0,0 +1,71 @@
+package kobe;
+
+import kobe.command.Command;
+import kobe.task.TaskList;
+import kobe.util.Parser;
+import kobe.util.Storage;
+import kobe.util.Ui;
+
+import java.io.IOException;
+
+/**
+ * The main class of the Kobe chatbot application.
+ * Handles the initialization of the application and runs the main program loop.
+ */
+public class Kobe {
+
+    /** Handles storage-related operations such as saving and loading tasks. */
```

Comment:
I like the detailed javadocs you provided for all variables and functions. Some hustle there!

Reply:
yeapp thank you, will keep it up!